src/controllers/interaction.ts

```diff
@@ -146,10 +126,12 @@ export const interactionEndControllerFactory =
         );
       }
 
+      req.session.authForProconnectFederation = undefined;
```

Comment:
suggestion: we should put this in a `clearInteractionSession()` function in `managers/session/interaction.ts`